README.md

```diff
@@ -1 +1,34 @@
-# hello_microverse-project
\ No newline at end of file
+<div align="center">
+
```

Comment:
- [ ] please kindly use the [Microverse README template](https://github.com/microverseinc/readme-template)
- [ ] please ensure All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. You are missing some. Like show your support, getting started, future features, contributing, acknowledgement which are all aimed at making your README professional. The only **optional** section is FAQ
- [optional] i will advise  get the whole README and edit, so you get the table of content, rather than fixing the old one(one you have) . Remember to remove the logo, also to remove the section description**, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project if you decide to edit the README. Nice work

Reply:
Hi @tufoinnkuo10 ! thank you for such a quick code review. Question regarding the markdown README file:

-1) tech stack: should I set the three options to 'none' and leave their respective -href- blank?

-2) regarding:  key-features , prerequisites, setup, install, usage, run test, deployment, features, contributing ------- can this be blank?

- 3) should I remove the => ***_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._***

Thank you so much for your time @tufoinnkuo10 !!

Reply:
yes you can remove the note and keep just the MIT link, stacks can put HTML and CSS, other parts put the necessary info as keeping them blank is not appropriate.